packages/devtools_shared/lib/src/server/server_api.dart

```diff
@@ -10,6 +10,7 @@ import 'dart:io';
 
 import 'package:shelf/shelf.dart' as shelf;
 
+import '../../devtools_shared.dart';
```

Comment:
import the file you need directly instead of importing the library that exports others